### PR TITLE
[plugin.video.eventvods@matrix] v1.0.1+matrix.1

### PR DESCRIPTION
**Pull request by Ikosse**
### Description
This addon makes it possible to browse and watch esports events from eventvods.com, which aims at providing a spoiler free experience. The addon extracts YouTube and Twitch links from eventvods json data and then uses the existing Youtube and Twitch addons from the official repository for playback.

### Add-on details:
* **General**
  * Add-on name: Eventvods
  * Add-on id: plugin.video.eventvods
  * Version number: 1.0.1+matrix.1
  * Kodi/repository version: matrix
* **Code location**
  * URL: https://github.com/Ikosse/plugin.video.eventvods

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.eventvods/addon.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="addon.py">
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en">Catch up on esports events completely spoiler free.</summary>
+        <summary lang="sv_SE">Få tillgång till de senaste esports-evenemangen helt utan spoilers.</summary>
+    	<description lang="en_GB">This is an unofficial addon that gives you access to the latests esports events from eventvods.com.</description>
+    	<description lang="sv_SE">Detta är ett inofficielt addon som ger dig tillgång till de senaste esports-evenemangen från eventvods.com.</description>
+        <license>GNU General Public License, v2</license>
```
Comment (enen92) [inline plugin.video.eventvods/addon.xml]: GPL-2.0-or-later or GPL-2.0-only (spdx identifier)

**Review by enen92 — CHANGES_REQUESTED** [review]
Just minors. Thanks for the submission

File: plugin.video.eventvods/addon.xml (hunk 1 above)
Comment (Ikosse) [reply plugin.video.eventvods/addon.xml]: Thank you for the feedback! I have modified the addon.xml and followed the instructions on https://kodi.wiki/view/HOW-TO:Update_a_pull_request_to_an_official_addon_repository_on_GitHub, I think. I'm still quite novice on git; is there something more I should do?

**Review by enen92 — APPROVED** [review]

File: plugin.video.eventvods/addon.xml (hunk 1 above)
Comment (enen92) [reply plugin.video.eventvods/addon.xml]: All good thanks

**Pull request merged**